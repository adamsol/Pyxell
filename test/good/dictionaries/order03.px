
d = {for i in 1..10 yield i: i %% 2}
print d.pop(), d.pop().b, d.pop().a
print [...d by 4]
