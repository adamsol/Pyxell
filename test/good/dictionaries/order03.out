(10, true) false 8
[(1, false), (5, false)]
